utils/scale-test

```diff
@@ -281,7 +281,7 @@ def Nelder_Mead_simplex(objective, params, bounds, epsilon=1.0e-6):
     locs = [tuple(random.uniform(*b) for b in bounds)
             for _ in range(ndim + 1)]
     SimplexPoint = namedtuple("SimplexPoint", ["loc", "val"])
-    simplex = [SimplexPoint(loc=l, val=f(l)) for l in locs]
+    simplex = [SimplexPoint(loc=L, val=f(L)) for L in locs]
```

Comment:
`loc` seems nicer than `l` or `L`.

Reply:
I thought of that, but worried that `loc=loc` wouldn't read right.  If you think it makes sense that way, I'll change it.